miner_exporter.py

```diff
@@ -41,7 +41,11 @@
                              ['resource_type'])
 LEDGER_PENALTY = prometheus_client.Gauge('validator_ledger',
                               'Validator performance metrics ',
-                             ['resource_type'])
+                             ['resource_type', 'subtype'])
+MINER_VERSION = prometheus_client.Info('miner_version',
```

Comment:
Grafana groups the first word in this string from what i can tell. wouldn't worry about it now but maybe we'll call them all miner? 
![image](https://user-images.githubusercontent.com/7544765/116159405-6771db00-a6ad-11eb-8da8-23754902a6a2.png)

Reply:
I haven't been putting it in graf yet. I'll change it so they stay together.